src/docs/Introduction.stories.mdx

```diff
@@ -0,0 +1,37 @@
+import { Meta, Story } from '@storybook/addon-docs';
+
+<Meta title="Introduction"/>
+
+<img src="/moja-global-icon.jpg" alt="logo" height="450" width="780"/>
```

Comment:
Please reduce the image's size.

Additionally, the changes suggested by @Crystalsage and @Janvi-Thakkar would be appropriate.